tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -1,12 +1,21 @@
 """Set permissions for users."""
+from collections import ChainMap
 
 from rest_framework import permissions
 
 
 def is_own_stt(request, view):
     """Verify user belongs to requested STT."""
     is_data_prepper = is_in_group(request.user, 'Data Prepper')
-    requested_stt = view.kwargs.get('stt', request.data.get('stt'))
+
+    # Depending on the request, the STT could be found in three different places
+    # sp we will merge all together and just do one check
+    request_parameters = ChainMap(
+        view.kwargs,
+        request.query_params,
```

Comment:
To make permissions work for the STT filter on the `list` endpoint we need to also include the query parameters supplied to the request here.